dd-sdk-android/src/main/kotlin/com/datadog/android/rum/GlobalRum.kt

```diff
@@ -143,6 +140,10 @@ object GlobalRum {
 
     // region Internal
 
+    internal fun checkSessionForUpdate() {
```

Comment:
This is not what this method does anymore, maybe we should rename this

Reply:
Yeah...I was trying to find a proper name, maybe `notifyIngestedWebViewEvent`